dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/deletefiles/DeleteItemPresenter.java

```diff
@@ -110,12 +117,11 @@ private void changeLabelText(final ObservableList<TreeItem<Ds3TreeTableValue>> s
     }
 
     public void deleteItems() {
-
         deleteTask.setOnCancelled(event -> constructMessageForLog());
         deleteTask.setOnFailed(event -> constructMessageForLog());
         deleteTask.setOnSucceeded(event -> {
             loggingService.logMessage(resourceBundle.getString("deleteSuccess"), LogType.SUCCESS);
-            LOG.info("Success to delete selected item(s):{} ", LogType.SUCCESS);
+            LOG.info("Successfully deleted selected item(s):{} ", LogType.SUCCESS);
```

Comment:
I know you didn't write this log message, but the `{}` doesn't tell us anything useful.  Remove that and the `LogType.SUCCESS`